slack-base/src/test/java/com/hubspot/slack/client/models/migration/MigrationExchangeParamsDeserializationTest.java

```diff
@@ -0,0 +1,64 @@
+import com.hubspot.slack.client.models.JsonLoader;
+import com.hubspot.slack.client.models.response.migration.MigrationExchangeResponse;
+
+public class MigrationExchangeParamsDeserializationTest {
+
+  private static final String VALID_USER_OLD_ID = "U1X3XFPJL";
+  private static final String VALID_USER_MIGRATED_ID = "U01SZDTN10E";
+  private static final String INVALID_USER_ID = "INVALID_ID";
+
+  @Test
+  public void itDeserializesMigrationExchangeResponseWithValidAndinvalidUserIds() throws IOException {
+    MigrationExchangeResponse migrationExchangeResponse =
+        fetchAndDeserializeMigrationExchangeResponse("migration_exchange_response_with_valid_and_invalid_user_ids.json");
+    Map<String, String> userIdMap = migrationExchangeResponse.getUserIdMap();
+    Set<String> invalidUserIds = migrationExchangeResponse.getInvalidUserIds();
+    assertTrue(doesMainParamsPresent(migrationExchangeResponse));
+    assertTrue(userIdMap.containsKey(VALID_USER_OLD_ID) && userIdMap.containsValue(VALID_USER_MIGRATED_ID));
+    assertTrue(invalidUserIds.contains(INVALID_USER_ID));
+  }
+
+  @Test
+  public void itDeserializesMigrationExchangeResponseWithOnlyValidUserIds() throws IOException {
+    MigrationExchangeResponse migrationExchangeResponse =
+        fetchAndDeserializeMigrationExchangeResponse("migration_exchange_response_with_only_valid_user_ids.json");
+    Map<String, String> userIdMap = migrationExchangeResponse.getUserIdMap();
+    Set<String> invalidUserIds = migrationExchangeResponse.getInvalidUserIds();
+    assertTrue(doesMainParamsPresent(migrationExchangeResponse));
+    assertTrue(userIdMap.containsKey(VALID_USER_OLD_ID) && userIdMap.containsValue(VALID_USER_MIGRATED_ID));
+    assertTrue(invalidUserIds.isEmpty());
+  }
+
+  @Test
+  public void itDeserializesMigrationExchangeResponseWithOnlyInvalidUserIds() throws IOException {
+    MigrationExchangeResponse migrationExchangeResponse =
+        fetchAndDeserializeMigrationExchangeResponse("migration_exchange_response_with_only_invalid_user_ids.json");
+    Map<String, String> userIdMap = migrationExchangeResponse.getUserIdMap();
+    Set<String> invalidUserIds = migrationExchangeResponse.getInvalidUserIds();
+    assertTrue(doesMainParamsPresent(migrationExchangeResponse));
+    assertTrue(userIdMap.isEmpty());
+    assertTrue(invalidUserIds.contains(INVALID_USER_ID));
+  }
+
+  private MigrationExchangeResponse fetchAndDeserializeMigrationExchangeResponse(String jsonFileName) throws IOException {
+    String rawJson = JsonLoader.loadJsonFromFile(jsonFileName);
+    return ObjectMapperUtils.mapper().readValue(rawJson, MigrationExchangeResponse.class);
+  }
+
+  private boolean doesMainParamsPresent(MigrationExchangeResponse migrationExchangeResponse) {
+    return migrationExchangeResponse.isOk() &&
+        !migrationExchangeResponse.getTeamId().isEmpty() &&
+        !migrationExchangeResponse.getEnterpriseId().isEmpty();
+  }
```

Comment:
I think it makes sense to change this to
```java
private void assertMainResponseFields(MigrationExchangeResponse migrationExchangeResponse) {
    assertThat(migrationExchangeResponse.isOk()).isTrue();
    assertThat(migrationExchangeResponse.getTeamId()).isEqualTo("T024G0P55");
    assertThat(migrationExchangeResponse.getEnterpriseId()).isEqualTo("E01L6J4CPS8");
  }
```
This way, we verify each field separately and will receive more meaningful error in case of failure. Also, we'll verify that the values from the json fixture are parsed correctly instead of simply checking that they are present.